### PR TITLE
[Docs] CLI command update for extracting the peer.

### PR DESCRIPTION
**Pull request by rajkaramchedu**
Updated docs for https://github.com/aptos-labs/aptos-core/pull/3885.

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/3899)

### CODE REVIEW

File: developer-docs-site/docs/nodes/full-node/network-identity-fullnode.md [hunk 1]
```diff
@@ -106,20 +106,63 @@ Follow the below detailed steps:
...
-   ```
-   $ aptos key extract-peer \
-       --private-key-file /path/to/private-key.txt \
-       --output-file /path/to/peer-info.yaml
-   ```
+  ```
+  aptos key extract-peer --host 1.1.1.1:6180 \
+      --public-network-key-file /path/to/private-key.txt.pub \
+      --output-file /path/to/peer-info.yaml
+  ```
```
Comment (sherry-x) [inline developer-docs-site/docs/nodes/full-node/network-identity-fullnode.md:165]: Need to update the example below for this section as well.

**Review by sherry-x — APPROVED** [review]

File: developer-docs-site/docs/nodes/full-node/network-identity-fullnode.md (hunk 1 above)
Comment (rajkaramchedu) [reply developer-docs-site/docs/nodes/full-node/network-identity-fullnode.md:165]: I didn't quite get that, @sherry-x? You mean the Docker case? I updated it?

Comment (gregnazario) [reply developer-docs-site/docs/nodes/full-node/network-identity-fullnode.md:165]: There's an output in the `peer-info` right below :)

**Review by gregnazario — APPROVED** [review]
Looks good, just fix the output of tghe addresses on line 174

File: developer-docs-site/docs/nodes/full-node/network-identity-fullnode.md (hunk 1 above)
Comment (sherry-x) [reply developer-docs-site/docs/nodes/full-node/network-identity-fullnode.md:165]: line 171 - 178

**Pull request merged**